governance/proposals/012-set-udt-swap-burner.js

```diff
@@ -0,0 +1,223 @@
+/**
+ * This proposal send calls accross the bridge to set swap burner
+ * in Unlock on Arbitrum, Base, and Optimism.
+ */
+const { ADDRESS_ZERO, getNetwork } = require('@unlock-protocol/hardhat-helpers')
+const { Unlock } = require('@unlock-protocol/contracts')
+const { IConnext } = require('../helpers/bridge')
+const { parseSafeMulticall, getProvider } = require('../helpers/multisig')
+const { ethers } = require('hardhat')
+const { Contract } = require('ethers')
+
+const targetChainsIds = [
+  42161, // Arbitrum
+  8453, // Base
+  10, // Optimism
+]
+
+// TODO: deploy swap burner contracts
+const swapBurnerAddresses = {
+  42161: '0x316A4650e70594FA3D947a43A237bEF427Bd80d6', // Arbitrum
+  8453: '0x440d9D4E66d39bb28FB58729Cb4D3ead2A595591', // Base
+  10: '0xd8250925527e769d90C6F2Fc55384B9110f26b62', // Optimism
+}
+
+const parseCalls = async (destChainId) => {
+  const { unlockAddress, unlockDaoToken, nativeCurrency, explorer } =
+    await getNetwork(destChainId)
+  const swapBurnerAddress = swapBurnerAddresses[destChainId]
+
+  const provider = await getProvider(destChainId)
+
+  // get Unlock interface
+  const unlock = new Contract(unlockAddress, Unlock.abi, provider)
+
+  if (BigInt(destChainId) !== (await unlock.chainId())) {
+    throw new Error(
+      `Mismatch btw ${destChainId} and Unlock settings ${await unlock.chainId()}`
+    )
+  }
+
+  // TODO: double check this number
+  const estimatedGasForPurchase = 20000n
```

Comment:
Currently we have it to `200000` on Ethereum mainnet
```suggestion
  const estimatedGasForPurchase = 200000n
```

Reply:
Here is seems that the estimation is a bit low. Checking (on Base, Arbitrum and Optimism) its more ~250k for native token an ~300k for a lock priced in ERC20 

ERC20
-https://optimistic.etherscan.io/tx/0x93396a5e5b39cd62a54320a7890dacdeb0033be6bf8e398194f967df4852e75b
- https://arbiscan.io/tx/0xc04af201ff709c22ba9dfbd6bb04b575391a2933d67f0834410bba15214e6dc4

native

- https://optimistic.etherscan.io/tx/0x743d13be4f858536d46d8fd7d378c39e105ce127a9f04c472d6819f51b87ce1f
- https://basescan.org/tx/0x724da794694c52659da14ba7d3f732f5e7aae2418ca89f37414d66a0148aebe3

Reply:
ok but then we need to change it everywhere? Also we need it to be just below the actual gas spent... so that nobody is abusing this?
If it's higher than the gas spent someone could "loop" buying a key from their own lock, receiving UDT, selling UDT ... etc.

So I would be in favor of keeping 200k for now.
I actually think we can change that later once the protocol fee is in place :) Let's discuss it soon!